lms/envs/aws.py

```diff
@@ -581,6 +581,8 @@
             'schedule': datetime.timedelta(hours=ENV_TOKENS.get('THIRD_PARTY_AUTH_SAML_FETCH_PERIOD_HOURS', 24)),
         }
 
+    SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS = ENV_TOKENS.get('SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS')
```

Comment:
Why is this setting called `SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS` here, while it's called simply `RESPECT_X_FORWARDED_HEADERS` in the rest of the code?